src/customTheme.ts

```diff
@@ -26,6 +30,8 @@ const primaryColor: ColorHues = {
   900: '#180319',
 }
 
+// The old blueColor palette will be replaced with a suitable hue out of the Sky TailwindCSS color palette.
+
 const blueColor: ColorHues = {
   50: '#def8ff',
```

Comment:
I just realized the current colors are defined incorrectly: the keys should be strings (`'50'`, `'100'`, etc) instead of numbers (`50`, `100`, etc). I'll create a small PR later to fix this.